app/js/lib/foxbox/db.js

```diff
@@ -13,15 +27,18 @@ const DB_TAG_STORE = 'tags';
 
 export default class Db {
   constructor() {
-    this.db = null;
+    // Private properties.
+    this[p.db] = null;
+
+    return Object.seal(this);
```

Comment:
OK, I'm going on with this approach (without the `return`) and we'll re-evaluate later if needed.